go/vt/schemadiff/schema.go

```diff
@@ -750,3 +759,258 @@ func (s *Schema) Apply(diffs []EntityDiff) (*Schema, error) {
 	}
 	return dup, nil
 }
+
+// TODO
```

Comment:
What does this `TODO` reference to?

Reply:
hehe, it references the fact that is is already done... removing...

Reply:
fixed